source/simulator/stokes_matrix_free.cc

```diff
@@ -755,6 +789,23 @@ namespace aspect
               for (unsigned int d=0; d<dim; ++d)
                 sym_grad_u[d][d] -= viscosity_x_2/3.0*div;
 
+            if (strain_rate_table !=nullptr
+                && !(strain_rate_table->empty()))
+              {
+                SymmetricTensor<2,dim,VectorizedArray<number>> grads_phi_u_i = velocity.get_symmetric_gradient (q);
+
+                // Note that derivative_scaling_factor * alpha is multiplied to
```

Comment:
Like this? Or do I misunderstand when this multiplication happens?
```suggestion
                // Note that derivative_scaling_factor * alpha has already been multiplied to
```